### ISSUE TITLE
Make existing debugging facilities for prototypes available through menu

### ISSUE THREAD

**Issue by RieksJ** [AmpersandTarski/Ampersand#1173]
Ampersand prototypes have a number of special API-calls that are very helpful for developers. There are API calls for exporting the database population, and for exporting a CSV file that reports on the time consumption of the various rules/conjuncts that have been evaluated, and I expect there to be more.

This issue calls for an extension of the prototype menu that already allows us to reinstall the database, i.e. to add items to that menu that, when selected, invoke such APIs.

**Comment by Michiel-s** [comment]
Agree. This will be implemented in the next gen frontend in Angular.
